packages/suite/src/views/settings/SettingsCoins/FirmwareTypeSuggestion.tsx

```diff
@@ -40,24 +43,32 @@ export const FirmwareTypeSuggestion = () => {
         dispatch(goto('settings-device', { anchor: SettingsAnchor.FirmwareType }));
 
     return (
-        <StyledCard>
-            <Description>
-                <Translation
-                    id={translationId}
-                    values={{
-                        button: chunks => (
-                            <StyledButton variant="tertiary" size="tiny" onClick={goToFirmwareType}>
-                                {chunks}
-                            </StyledButton>
-                        ),
-                        bitcoinOnly: <Translation id="TR_FIRMWARE_TYPE_BITCOIN_ONLY" />,
-                        regular: <Translation id="TR_FIRMWARE_TYPE_REGULAR" />,
-                    }}
-                />
-            </Description>
-            <Button variant="tertiary" size="small" onClick={handleClose}>
-                <Translation id="TR_GOT_IT" />
-            </Button>
-        </StyledCard>
+        <Container>
+            <Box variant="primary">
+                <Row>
+                    <Description>
```

Comment:
This is so many indentations that my head feels dizzy :D

What about extracting the `<Description....` part into a separate component so its a less of a spaghetti?

Reply:
Done